qt/applications/workbench/CMakeLists.txt

```diff
@@ -28,10 +28,6 @@ set ( TEST_FILES
   workbench/widgets/plotselector/test/test_plotselector_view.py
 )
 
-set ( WORKBENCH_TEST_PY_FILES
```

Comment:
Just for future reference: we discussed this offline and these tests have been too flaky to continue being supported. the balance of unreliable tests vs manual testing of some components came out in favour of more robust automated tests.